engine/src/flutter/impeller/display_list/aiks_dl_text_unittests.cc

```diff
@@ -155,6 +155,51 @@ TEST_P(AiksTest, CanRenderTextFrameWithHalfScaling) {
   ASSERT_TRUE(OpenPlaygroundHere(builder.Build()));
 }
 
+TEST_P(AiksTest, Subpixel) {
+  DisplayListBuilder builder;
+
```

Comment:
These golden tests probably need a content scalar, it isn't in right now because of that local testing bug.